app/views/publishers/vacancies/build/applying_for_the_job.html.slim

```diff
@@ -23,11 +23,16 @@
             - else
                 = f.govuk_url_field :application_link, label: { size: "s" }, form_group: { classes: "optional-field" }
           - else
-            = govuk_notification_banner title: t("banners.important") do |banner|
-              - banner.add_heading text: t(".notification.heading")
-              = t(".notification.description.content_html", link: govuk_link_to(t(".notification.description.link"), page_path("job-application-preview"), target: "_blank"))
+            - if current_variant?(:"2021_04_application_preview_test", :banner_link)
+              = govuk_notification_banner title: t("banners.important") do |banner|
+                - banner.add_heading text: t(".banner.heading")
+                = t(".banner.description.content_with_link_html", link: govuk_link_to(t(".banner.description.link"), page_path("job-application-preview"), target: "_blank"))
```

Comment:
If it's only the body changing between variants, you could have the conditional inside the notification banner